charts/thoras/templates/api-server-v2/deployment.yaml

```diff
@@ -118,6 +118,10 @@ spec:
                 fieldPath: metadata.namespace
         ports:
         - containerPort: {{ .Values.thorasApiServerV2.containerPort }}
+        readinessProbe:
```

Comment:
What is the health endpoint doing? Is it making a round-trip ping to the DB? that would be ideal for a readiness probe. Anything less should probably be a liveness probe